plotview.cpp

```diff
@@ -382,20 +382,27 @@ void PlotView::setFFTAndZoom(int size, int zoom)
 
 void PlotView::setPowerMin(int power)
 {
-    powerMin = power;
+//  HVI_REVIEW: Not needed?   powerMin = power;
```

Comment:
It does look like these are unused and should be removed. I'd prefer that as a separate commit that just deletes the lines, rather than commenting them out